pkg/kopia/cli/internal/flag/storage/model/location_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2024 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"testing"
+
+	rs "github.com/kanisterio/kanister/pkg/secrets/repositoryserver"
+	"gopkg.in/check.v1"
+)
```

Comment:
```suggestion
       "gopkg.in/check.v1"
       
	rs "github.com/kanisterio/kanister/pkg/secrets/repositoryserver"
	
)
```

We pkg imports into 3 groups:
built-in packages, 3rd party imports and then kanister

Reply:
Okay, I will do it. Is it possible to implement this at the linter level?